hamp_radar/iquick.py

```diff
@@ -234,6 +233,157 @@ def postprocess_iq(ds):
+                        ),
+                        "cx_NPw": radar["NPw"]
+                        .isel(cocx=1)
+                        .assign_attrs(
+                            {"long_name": "cross-channel power from calibration source"}
+                        ),
+                        "co_CPw": radar["CPw"]
+                        .isel(cocx=0)
+                        .assign_attrs(
+                            {"long_name": "co-channel power from noise source"}
+                        ),
+                        "cx_CPw": radar["CPw"]
+                        .isel(cocx=1)
+                        .assign_attrs(
+                            {"long_name": "cross-channel power from calibration source"}
+                        ),
+                    },
+                )
+                .assign_coords(
+                    radar_time=(radar.time).assign_attrs(
+                        {
+                            "long_name": "timestamp for frame",
+                            "provenance": "assigned by radar",
+                        }
+                    )
+                )
+                .assign_coords(
+                    frame=(radar.frm).assign_attrs(
+                        {
+                            "long_name": "timestamp for frame",
+                            "method": "assigned by radar",
+                        }
+                    )
+                )
+            )
+            ds1.to_zarr(
+                f"{zarr_path}/HALO-{pd.to_datetime(radar.time[0].values):%Y%m%da-%H%M%S}-frms.zarr"
+            )
+
+            ds2 = (
+                xr.Dataset(
+                    {
+                        "co": radar["FFTD"]
+                        .isel(cocx=0)
+                        .assign_attrs(
+                            {"long_name": "iq data from co-polarazied receiver"}
+                        ),
+                        "cx": radar["FFTD"]
+                        .isel(cocx=1)
+                        .assign_attrs(
+                            {"long_name": "iq data from cross-polarazied receiver"}
+                        ),
+                    },
+                    coords={
+                        "range": (
+                            ("range",),
+                            (wait1_dt + np.arange(ngate) * pulse_dt) * c_air * 0.5,
+                            {
+                                "long_name": "range",
+                                "units": "m",
+                                "provenance": "estimated using pulse length",
+                            },
+                        ),
+                        "iq": (
+                            ("iq",),
+                            ["i", "q"],
+                            {"long_name": "signal phase (inphase or quadrature)"},
+                        ),
+                    },
+                )
+                .assign_coords(
+                    time=(time_frame + dtfft).assign_attrs(
```

Comment:
Why would you assign the `time` coord later on? It could also be with the other coords in the constructor.

Reply:
my first attempts failed because time was a coordinate with a different meaning before the stack, and only went away thereafter... but probably I missed a better way to fix this